examples/svelte-kit-demo/svelte.config.cjs

```diff
@@ -5,6 +5,8 @@ module.exports = {
 		// specifying a different adapter
 		adapter: '@sveltejs/adapter-node',
 
+		clientSideRouting: false,
```

Comment:
I haven't looked at the rest of the PR, but we almost certainly don't want this to be the default for any of the general demos.

Reply:
ah yup, accidentally left in during testing, fixed below